stable/airflow/templates/scheduler/scheduler-deployment.yaml

```diff
@@ -261,6 +261,24 @@ spec:
               {{- end }}
                && echo "*** running scheduler..." \
                && exec airflow scheduler -n {{ .Values.scheduler.numRuns }}
+          livenessProbe:
```

Comment:
don't you want to disable it by default?

Reply:
It's definitely an important feature, I don't think anyone should be disabling it.

Reply:
I am just afraid if it fails for some user for unknown reason. I would prefer an opt-in feature on this one, is it possible?

Reply:
@gsemet I have now added an enable flag.

I have set it to default as `true`, because everyone should really be using this feature.

Reply:
Yes I agree, but I know environments can be different, and liveness probe are quite hard to tune. So ok to set it on by default, just add the node to disable it in the README and will be good to go.

Reply:
Since I cant think of a situation which wold require a user to disable it, I have no idea what I would add to the README.md. 

I think it should be fairly obvious to users how to disable it, (especially if they are in the extreme edge case that needs to disable it).

So, are you ok to merge this without the README.md change? Users seem to really want this feature, (especially as airflow scheduler sometimes just dies in normal operation in large clusters).

Reply:
@thesuperzapper the liveness probe failed for me:

It seems like an issue for the environment path:

```
    Liveness:  exec [python -Wignore -c import os
os.environ['AIRFLOW__CORE__LOGGING_LEVEL'] = 'ERROR'
os.environ['AIRFLOW__LOGGING__LOGGING_LEVEL'] = 'ERROR'
from airflow.jobs.scheduler_job import SchedulerJob
from airflow.utils.net import get_hostname
import sys
job = SchedulerJob.most_recent_job()
sys.exit(0 if job.is_alive() and job.hostname == get_hostname() else 1)
]
```
I can not `import airflow` like this ^^, which results in this

```
  Warning  Unhealthy    6m5s (x31 over 7h10m)  kubelet, minikube  Liveness probe failed: Traceback (most recent call last):
  File "<string>", line 4, in <module>
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/__init__.py", line 31, in <module>
    from airflow.utils.log.logging_mixin import LoggingMixin
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/utils/__init__.py", line 24, in <module>
    from .decorators import apply_defaults as _apply_defaults
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/utils/decorators.py", line 36, in <module>
    from airflow import settings
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/settings.py", line 37, in <module>
    from airflow.configuration import conf, AIRFLOW_HOME, WEBSERVER_CONFIG  # NOQA F401
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/configuration.py", line 731, in <module>
    conf.read(AIRFLOW_CONFIG)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/configuration.py", line 421, in read
    self._validate()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/configuration.py", line 213, in _validate
    self._validate_config_dependencies()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/configuration.py", line 245, in _validate_config_dependencies
    raise AirflowConfigException(
airflow.exceptions.AirflowConfigException: error: cannot use sqlite with the KubernetesExecutor
```

Reply:
```
kubectl -n airflow exec -it airflow-scheduler-5bb8678846-pnrrq -c airflow-scheduler -- bash -l -c "python -Wignore -c \"import os; os.environ['AIRFLOW__CORE__LOGGING_LEVEL'] = 'ERROR'; os.environ['AIRFLOW__LOGGING__LOGGING_LEVEL'] = 'ERROR'; from airflow.jobs.scheduler_job import SchedulerJob; from airflow.utils.net import get_hostname; import sys; job = SchedulerJob.most_recent_job(); print(job.is_alive(), job.hostname, get_hostname());\""
True airflow-scheduler-5bb8678846-pnrrq airflow-scheduler-5bb8678846-pnrrq
```

Adding `bash -l` will fix the issue, but I am not sure if this would cause issues for others